src/javascript/app_2/App/Components/Elements/DataTable/table_row.jsx

```diff
@@ -1,31 +1,43 @@
-import React     from 'react';
 import PropTypes from 'prop-types';
+import React     from 'react';
 import TableCell from './table_cell.jsx';
 
-const TableRow = ({ columns, row_obj = {}, is_header, is_footer }) => (
-    <div className='table__row'>
-        {columns.map(({ col_index, renderCellContent, title }) => {
-            const cell_value = row_obj[col_index] || '';
+const TableRow = ({
+    columns,
+    is_footer,
+    is_header,
+    onRowClick,
+    row_obj = {},
+}) => {
+    const onClick = onRowClick ? (() => { onRowClick(row_obj); }) : null;
 
-            return (
-                <TableCell col_index={col_index} key={col_index}>
-                    {is_header
-                        ? title
-                        : renderCellContent
-                            ? renderCellContent({ cell_value, col_index, row_obj, is_footer })
-                            : cell_value
-                    }
-                </TableCell>
-            );
-        })}
-    </div>
-);
+    return (
+        <div className='table__row' onClick={onClick}>
```

Comment:
maybe we use `RedirectOnClick` component here? [Like for portfolio drawer card](https://github.com/binary-com/binary-static/pull/4146/files#diff-9817baa1d2dd1ac4bd3ebff18bbe0decR9). This way I guess we don't need to pass any additional props.

Reply:
The reason I didn't use it here was that `DataTable` is a common component, using `RedirectOnClick` here will prevent it from doing other things rather than redirecting (i.e. displaying a popup). So, I used a more general way for this to cover all needs.